rustfmt.toml

```
@@ -1,5 +1,6 @@
 reorder_imports = true
 imports_granularity = "Crate"
+group_imports = "One"
```

```suggestion
group_imports = "StdExternalCrates"
```

works for me

I'd also like to go unify this across the rest of the repos in this family